docs/en/observability/ci-cd-observability.asciidoc

```diff
@@ -150,6 +150,80 @@ To investigate further, you can view the details of the build captured as labels
 .Contextual attributes of a Jenkins pipeline execution in Elastic Observability
 image::images/jenkins-pipeline-context.png[Attributes of a Jenkins pipeline execution]
 
+[discrete]
+[[ci-cd-store-jenkins-logs]]
+== Store Jenkins pipeline logs in Elastic
+
+Jenkins pipeline logs can be sent through OpenTelemetry Protocol (OTLP) to be stored
+in an observability backend alongside the traces of the pipeline builds and Jenkins health metrics.
+Storing logs in an observability backend provides several benefits including:
+
+* Better observability, monitoring, alerting, and troubleshooting of the Jenkins instance
+thanks to the unification of all the signals in the observability backend.
+* Better traceability and control of long term storage for builds so you can more easily
+audit the Software Delivery Lifecycle.
+* Better scalability and reliability of Jenkins by greatly reducing the quantity of data stored in
+Jenkins and limiting the well known file system performance challenges of Jenkins when storing a large history of builds.
+
+[discrete]
+[[ci-cd-store-jenkins-logs-requirements-configuration]]
+=== Requirements and configuration
+
+Storing Jenkins pipeline logs in Elastic requires:
+
+* Elastic Observability version 8.1 or higher.
+* The OpenTelemetry Protocol endpoint configured on the Jenkins OpenTelemetry Plugin must be
+reachable from the Jenkins Agents (i.e. don't specify a localhost OTLP endpoint unless OpenTelemetry
+collectors are also deployed on the Jenkins Agents).
+* When using OpenTelemetry Collectors, set up a https://opentelemetry.io/docs/collector/configuration/#service[logs pipeline]
+in addition to the traces and metrics pipelines.
+
+To store pipeline logs in Elastic:
+
+1. Navigate to the _OpenTelemetry_ section of Jenkins configuration screen.
+2. Set the _OTLP Endpoint_.
+3. Use the _Add Visualisation Observability Backend_ dropdown to select the *Elastic Observability* option.
+4. Set the _Kibana base URL_.
+5. Click the _Advanced_ button to choose a storage integration strategy.
+There are two options for storing pipeline logs in Elastic Observability:
+    * *Store pipeline logs in Elastic and visualize logs both in Elastic and through Jenkins*,
+    which means you can view logs stored in Elastic on demand in the Jenkins UI.
+    Read more in <<ci-cd-visualize-logs-kibana-and-jenkins>> below.
+    * *Store pipeline logs in Elastic and visualize logs exclusively in Elastic*,
+    which means logs will no longer be visible through the Jenkins UI.
+    Read more in <<ci-cd-visualize-logs-kibana>> below.
+
+[discrete]
+[[ci-cd-visualize-logs-kibana-and-jenkins]]
+=== Visualize logs in both Kibana and Jenkins
+
+The Jenkins OpenTelemetry also provides pipeline log storage in Elasticsearch while enabling you to
```

Comment:
Not a big deal, but we try to use attributes for product names when possible. You might consider replacing Elasticsearch with `{es}` and Kibana with `{kib}`. See https://github.com/elastic/docs/blob/master/shared/attributes.asciidoc#L211-L212.

Reply:
Thanks @bmorelli25 -- has the team (or another docs team) considered implementing tests that assert that contributors consistently using these attributes?

Reply:
```suggestion
The Jenkins OpenTelemetry plugin also provides pipeline log storage in Elasticsearch while enabling you to
```

Reply:
> Thanks @bmorelli25 -- has the team (or another docs team) considered implementing tests that assert that contributors consistently using these attributes?

No, but that's a great suggestion for the new docs system!